modular_robot/revolve2/modular_robot/body/v2/_core_v2.py

```diff
@@ -0,0 +1,70 @@
+from pyrr import Vector3
+
+from .._color import Color
+from .._right_angles import RightAngles
+from ..common import Core
+
+
+class CoreV2(Core):
+    """The core module of a modular robot."""
+
+    _BATTERY_MASS = 0.39712  # in kg
+    _FRAME_MASS = 1.0644  # in kg
+
+    color = Color(255, 50, 50, 255)
+    _horizontal_offset = 0.029
+    _vertical_offset = 0.032
+    _attachment_positions: list[int]  # TODO: make this into meta-class
```

Comment:
I don't understand what this variable does? can you explain

Reply:
i made a comment next to it. its basically for the core to know on which position children should be attached.